src/styles/CompareCards.ts

```diff
@@ -187,8 +187,8 @@ export const SearchStyles = (mode: string) => {
       left: '220px',
       $nest: {
         '&.big': {
-          left: '190px',
-          minWidth: '510px',
+          left: '212px',
+          minWidth: `${850 - repoDropdownWidth - Spacing.Default}px`,
```

Comment:
note: I looked at the style of this recently, I was surprised how the style worked (with some absolute positioning etc instead of flex). I thought this was because of the dropdown. But let's not change that now anyway.

Reply:
Yes, we can deal with this in a separate cleanup PR.